purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -391,12 +394,9 @@ class PostingTransactionsTests {
     @Test
     fun `customer info cache is updated when purchasing`() {
         postReceiptSuccess = PostReceiptCompletionContainer()
-        val productIds = listOf("uno", "dos")
-        val purchase =
-            stubGooglePurchase(productIds = productIds).toStoreTransaction(ProductType.SUBS, null)
 
         underTest.postToBackend(
-            purchase = purchase,
+            purchase = mockk(relaxed = true),
```

Comment:
Using `relaxed=true` has lead me to unexpected results in the past. I consider a better practice to stub the whole object like we were doing, so I think it's better to do what we were doing before (stubbing the store transaction). Is there any reason why you changed this to a relaxed mock? I see we are using other relaxed mocks in this file which I would like to avoid but that's not part of this PR

Reply:
i was getting some tricky-to-debug failures that seemed unrelated to the test, and figured i could follow suit from other tests. but i'll try again now that it's been a few days, perhaps i can figure it out after the break